Automattic/wp-calypso
# Performance Profiler: Add filtering to the recommendations section

Fixes https://github.com/Automattic/dotcom-forge/issues/9049

## Proposed Changes

Add a dropdown with options to filter the recommendations by metric.

![CleanShot 2024-09-11 at 09 51 29@2x](https://github.com/user-attachments/assets/b9a66425-5ccd-4fcd-8b8e-399cada65f43)

## Why are these changes being made?

Fixes https://github.com/Automattic/dotcom-forge/issues/9049

## Testing Instructions
* Go to `speed-test-tool?url=https%3A%2F%2Fwordpress.com%2F&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NzQ2N30.LOelpUBxFDxMytpSYeB3EBNdBnHeYTvonyZ6zE7aqSM`
* Check if the dropdown is displayed on the Recommendations section
* Click in the list and see if the filtering is done.
